lib/testutils/auth.go

```diff
@@ -405,3 +405,6 @@ func (c *AuthClient) UploadSessionRecording(r events.SessionRecording) error {
 func (c *AuthClient) GetClusterCACert() (*auth.LocalCAResponse, error) {
 	return nil, nil
 }
+func (c *AuthClient) GenerateUserCerts(key []byte, user string, ttl time.Duration, compatibility string) (ssh []byte, tls []byte, err error) {
```

Comment:
For some reason, the [revision](https://github.com/gravitational/teleport/commit/0fd17) this branch was referring to had been deleted (probably with the sub-branch or something similar) hence I had to update the teleport's revision as well which introduced this change.